### PR TITLE
Add support for Image-Charts as a drop-in-replacement for Google Image Charts

### PR DESCRIPTION
**Pull request by FGRibreau**
👍

### CODE REVIEW

File: src/Leg/GCharts/BaseChart.php [hunk 1]
```diff
@@ -16,10 +16,24 @@
 
 class BaseChart implements ChartInterface
 {
+
+  /**
+   * Currently support charts by Image Charts
+   * @link https://image-charts.com/swagger.json
+	 * @var array
```
Comment (tgalopin) [inline src/Leg/GCharts/BaseChart.php]: There is indentation issues in this file, here is the first.

File: src/Leg/GCharts/BaseChart.php [hunk 2]
```diff
@@ -16,10 +16,24 @@
...
+	 */
+  const IMAGECHARTS_SUPPORTED = ['bvs', 'bhs', 'bvg', 'bhg', 'bvo', 'p', 'p3', 'pc', 'pd', 'ls', 'lc', 'lxy'];
```
Comment (tgalopin) [inline src/Leg/GCharts/BaseChart.php]: Same here.

File: src/Leg/GCharts/BaseChart.php [hunk 3]
```diff
@@ -16,10 +16,24 @@
...
+
 	/**
 	 * @var string
 	 */
-	const BASE_URL = 'http://chart.googleapis.com/chart';
+	const IMAGECHART_BASE_URL = 'http://image-charts.com/chart';
+
+  /**
```
Comment (tgalopin) [inline src/Leg/GCharts/BaseChart.php]: Same here.

**Review by tgalopin — CHANGES_REQUESTED** [review]
Thanks! That's cool to see a replacement for Google Charts image.

**Comment by FGRibreau** [comment]
@tgalopin thanks for bringing that up, fixed! :)

**Review by tgalopin — APPROVED** [review]

**Pull request merged**